src/libraries/System.Reflection/tests/TypeInfoTests.cs

```diff
@@ -490,6 +491,12 @@ public void IsEnumDefined_Invalid()
         [InlineData(typeof(CompoundClass4<string>), new Type[] { typeof(GenericInterface1<string>), typeof(TI_NonGenericInterface1) })]
         public void ImplementedInterfaces(Type type, Type[] expected)
         {
+            if (PlatformDetection.IsMonoRuntime)
+            {
+                // Mono re-orders interfaces due to #40593 and it's not clear why
+                return;
+            }
+
```

Comment:
```suggestion
```

Reply:
Oh, does the attribute suppress running on Mono?

Reply:
Yup, with ActiveIssueAttribute in place using TestRuntimes.Mono, this condition was redundant.